packages/edit-post/src/store/selectors.js

```diff
@@ -287,3 +287,14 @@ export function hasMetaBoxes( state ) {
 export function isSavingMetaBoxes( state ) {
 	return state.metaBoxes.isSaving;
 }
+
+/**
+ * Returns the current editing canvas device type.
+ *
+ * @param {Object} state Global application state.
+ *
+ * @return {string} Device type.
+ */
+export function __experimentalGetPreviewDeviceType( state ) {
```

Comment:
Would it make sense to add a comment explaining why the API is experimental? E.g: it may need to be moved to a package where it can be consumed by edit-site or edit post etc?